### ISSUE TITLE
A pwa demo

### ISSUE THREAD

**Issue by libreom** [captn3m0/epicqr#2]
https://whatpwacando.today/barcode

**Comment by captn3m0** [comment]
I've pushed the browser build: https://github.com/captn3m0/epicqr/blob/main/src/lib.js

It is relatively simple to use, with a single script tag, and the following snippet

```js
new eci.EpicQR.decode("dbhvecY6Roa4NF3gAzEbkTibZZzXAEYpMg8197BQWMS2+ID24FGDKWB5IEcuxjsA81ChprhSO3EsjKMRDbBWLg==")
// Object { epic: "ABC1234566", id: 1234 }
```

If someone is interested, I'd appreciate a PR to the (orhpan) gh-pages branch, with a script tag pointing to `https://raw.githack.com/captn3m0/epicqr/main/src/lib.js`.

Doesn't have to be fancy, just show an alert with the EPIC details.

**Comment by libreom** [comment]
I will try👍,It will only work in chrome, safari(ios17+) and not firefox android, can switch to a cross browser library later

**Comment by libreom** [comment]
Hi, I created a naive demo https://libreom.github.io/epicqr/ using https://github.com/zxing-js/library library, we can switch to native barcode detection api later but it isn't present in all browsers and only supports decoding(if you plan to add encoding too, I'd say disclose first), I'll add "choose image" button next . Try with this 
![image](https://github.com/captn3m0/epicqr/assets/111110599/ec26a8b4-4978-4dbb-9a40-9ef9294b42b3)
Also create a orphan branch so I can draft pr easily

**Comment by captn3m0** [comment]
I do plan on creating something for encoding, but I don't think it belongs here.

I've created a orphan `gh-pages` branch with an empty commit, you can file a PR against that. I tested the demo on Safari/Firefox+iOS and it didn't work for me. Could be lockdown mode, but I'm not sure.

**Comment by libreom** [comment]
Worked on chrome? Maybe try with image on previous comment as I was trying with sample image on your repo and it wasn't working

**Comment by captn3m0** [comment]
Tested on Brave/iOS and Chrome/iOS as well. The camera doesn't trigger at all.

**Comment by captn3m0** [comment]
Yeah, webcam test reports no camera devices on the iPhone. Looks like Lockdown mode breaks [MediaDevices.getUserMedia()](https://alexi.sh/blog/2022/07/lockdown-jsc/#mediadevicesgetusermedia) intentionally, so this is expected behaviour. I'll try on other devices, and report back.

**Comment by captn3m0** [comment]
Suggestion: Report no camera, and offer a paste mode to let the user paste a QR code string. 

Downgrade to an experience that still works :)

**Comment by libreom** [comment]
Ok, I'll add a "choose image" button for decoding already taken image and also a paste qr text textarea
